src/js/utils/dom.js

```diff
@@ -607,6 +609,33 @@ export function findPosition(el) {
  *
  */
 export function getPointerPosition(el, event) {
+  const translated = {
+    x: 0,
+    y: 0
+  };
+
+  if (browser.IS_IOS) {
```

Comment:
on iOS, iterate up the DOM tree and add up all the translation transforms so that we can modify the pageX and pageY coordinates for changedTouches to account for it.